### ISSUE TITLE
Change Hard-coded Nodes

### ISSUE THREAD

**Issue by matthewjamesr** [MyHush/hush#78]
# Issue
We need to adjust the hard-coded nodes to account for removal of myhush.network.

# Replacement list
* explorer.myhush.org
* hushipv4.matthewreichardt.com

**Comment by leto** [comment]
explorer.myhush.org is behind CloudFlare and can't be used. We should choose a few TLDs that span different countries to prevent one country being able to mess with us via root DNS servers

**Comment by acheven** [comment]
Is this why fresh install of hush-1.0.12.msi is not syncing?

**Comment by matthewjamesr** [comment]
@acheven Yes that is correct. We will be adding the nodes that get outputted from our updated config generator.

https://myhush.github.io/config-generator/

**Comment by mokio8** [comment]
@matthewjamesr how about exposing the peersList from config-generator as an api endpoint to make it easier for those of us with external applications (including this repo) to ping instead of duplicating efforts?

**Comment by matthewjamesr** [comment]
@mokio8 I can do that for third-party for sure. I do not know if we would want that for `hushd` though. Can you please open your recommendation as an issues in that repo? Thanks

**Comment by matthewjamesr** [comment]
@mokio8 I just added it.

https://raw.githubusercontent.com/MyHush/config-generator/master/peers.json

It should resolve to the following address in time:

https://myhush.github.io/config-generator/peers.json

**Comment by mokio8** [comment]
Thank you @matthewjamesr! As for hushd are you concerned at how often that endpoint might get hit and the potential latency in front of it?

**Comment by matthewjamesr** [comment]
We can do that @mokio8. Let's do `seed.myhush.org` -> `https://myhush.github.io/config-generator/peers.json`
